estate/models/estate_property.py

```diff
@@ -0,0 +1,22 @@
+from odoo import fields, models
+
+class TestModel(models.Model):
+    _name = "estate_property"
+    _description = "estate property descripction"
+
+    name = fields.Char(required=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date()
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float()
+    bedrooms = fields.Integer()
+    living_area = fields.Integer()
+    facedes = fields.Integer()
+    garage = fields.Boolean()
```

Comment:
Always set the label of the field. even if it's obvious and even if the orm will generate one for you using the field name.
postcode = fields.Char("Postcode") or postcode = fields.Char(string="Postcode")
-> better the first in this case, unless string is not the first param. (typically for relational fields)